Makefile

```diff
@@ -44,9 +68,19 @@ image/push:
 .PHONY: image/build/push
 image/build/push: image/build image/push
 
-.PHONY: cluster/prepare/local
-cluster/prepare/local:
-	-oc create -f deploy/crds/*_crd.yaml
-	@oc create -f deploy/service_account.yaml
-	@oc create -f deploy/role.yaml
-	@oc create -f deploy/role_binding.yaml
+.PHONY: image/build/test
+image/build/test:
+	operator-sdk build --enable-tests $(REG)/$(ORG)/$(PROJECT):$(TAG)
+
+.PHONY: cluster/prepare
+cluster/prepare:
```

Comment:
The only thing that I'm aware of that needs to be done in prepare for the e2e tests is create the namespace https://github.com/integr8ly/integreatly-operator/blob/master/Makefile#L112-L116

Reply:
I can split this out. The cluster/prepare was already there. It's needed for running things locally. I can add a seperate one for prow